frontend/src/bundles/auth/pages/auth.tsx

```diff
@@ -88,10 +89,10 @@ const Auth: React.FC = () => {
     }, [isPasswordForgot]);
 
     useEffect(() => {
-        if (dataStatus === DataStatus.FULFILLED) {
+        if (user) {
```

Comment:
why do we need these changes?

Reply:
@nikita-remeslov  Because we get fulfilled status for logout too.
And accordingly, we are then returned to Root
From Root, if !userAuthenticated, we are redirected to SignIn
And on SignIn - the status is still fulfilled ....
So we have looping and redirecting again and again

Reply:
but It's not async operation and you are not receiving any data, so probably you don't even need to change this state in case of logout action

Reply:
```
 builder.addCase(logout.pending, (state) => {
        state.dataStatus = DataStatus.PENDING;
});

builder.addCase(logout.fulfilled, (state) => {
        state.user = null;
        state.dataStatus = DataStatus.FULFILLED;
 });

builder.addCase(logout.rejected, (state) => {
        state.dataStatus = DataStatus.REJECTED;
});
```

Sorry, @dimapopovych. Maybe I don't understand something well.
But we need to remove the user from Redux.

If I remove this line `state.dataStatus = DataStatus.FULFILLED;`
then the status will remain PENDING

Or how should I do it?

![Screenshot_pending](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/106997950/fed4b591-9aba-41ad-8a9b-509830dfbfef)

Reply:
You could set the data status to pending instead of fulfilled, mimicking the initial state. However, I don't think it's worth it. It seems fine to rely on user presence to perform the redirect.

Reply:
I meant idle, sorry. But yeah, I think you could just keep this condition